src/Uno.UWP/Globalization/ApplicationLanguages.cs

```diff
@@ -79,29 +110,6 @@ private static void ApplyLanguages()
 
 				Languages = languages.Distinct().ToArray();
 			}
```

Comment:
Can we add back logging for when primarylanguage changes, and perhaps on startup when culture is being set

Reply:
Sure. While adding back the logging, I found out that I no longer fallback to `Languages[0]` when `PrimaryLanguageOverride` is empty string.

I'm not sure what is WinUI behavior around that, but I preserved back the original Uno behavior regarding this specific point.